prewl/endpoints/hf.py

```diff
@@ -22,9 +23,13 @@ def _call_remote(self, prompt):
         assert 'token' in CONFIG['backend'], "Must provide 'token' in the 'backend' configuration to call huggingface remotely."
         token = CONFIG['backend']['token']
 
-        params = {
-            'max_new_tokens': min(250, CONFIG.get('max_length', CONFIG['defaults']['max_length']))
-        }
+
+        if CONFIG['backend']['service'] == 'bert-base-uncased':
+            params = {}
+        else:
+            params = {
+                'max_new_tokens': min(250, CONFIG.get('max_length', CONFIG['defaults']['max_length']))
+            }
```

Comment:
I was a little unclear in future, how many different sets of parameters there would be. If there may be a lot, or some complex parameter lists, then moving to another file like the parsers is a good idea, but for now or if there is not going to be, then an if statement seems okay.